cmd/earthly/satellite_cmds.go

```diff
@@ -162,6 +162,10 @@ func (app *earthlyApp) actionSatelliteLaunch(cliCtx *cli.Context) error {
 	app.console.Printf("Launching Satellite. This could take a moment...\n")
 	err = cloudClient.LaunchSatellite(cliCtx.Context, app.satelliteName, orgID)
 	if err != nil {
+		if errors.Cause(err) == context.Canceled {
+			app.console.Printf("Operation cancelled. Satellite will finish launching in background.\n")
```

Comment:
The wording here sounds a bit odd, it's almost as if it says "Operation cancelled, Operation will continue (in the background)".

Furthermore, there's no guarantee the satellite will actually launch -- the user could be hung waiting for a TCP handshake before making the launch request.

maybe this would be more accurate:

```suggestion
			app.console.Printf("Exiting due to interrupt signal; Satellite launch will continue (if request was successfully received by the server)\n")
```

Reply:
Yeah that's true. I'll change the wording. Eventually, we could make the process more robust, perhaps by the server acknowledging that it's started the process. In typical cases this should work though, I almost never wait for the operations to finish as it is 😅